docs/pages/elevation.js

```diff
@@ -0,0 +1,166 @@
+// @flow strict
+import { type Node } from 'react';
+import { Box, Flex, Text } from 'gestalt';
+import MainSection from '../components/MainSection.js';
+import PageHeader from '../components/PageHeader.js';
+import Page from '../components/Page.js';
+
+export default function ColorUsagePage(): Node {
+  return (
+    <Page title="Elevation">
+      <PageHeader
+        name="Elevation"
+        description="The elevation allows elements to be lifted from the background and can be defined by applying drop shadows or other visual cues, such as color and borders."
+        showSourceLink={false}
+      />
+      <MainSection
+        name="Elevation using shadows"
+        description="Shadows reflect a spatial relationship by indicating separation from the background or allowing content to be scrolled under another element. Elevation using shadows includes two levels: floating and raised."
+      >
+        <MainSection.Subsection
+          title="Floating"
+          description="The default elevation level.
+        Elevates messages temporarily appearing in front of other surfaces, such as modals and banners. "
+        >
+          <Flex gap={4} direction="column">
+            <Box width={300} height={300} rounding={3} borderStyle="shadow" />
+            <Text>$elevation-floating</Text>
+            <Text>10% (#000000) opacity / Blur: 8</Text>
+          </Flex>
+          <MainSection.Subsection columns={2}>
+            <MainSection.Card
+              cardSize="md"
+              type="do"
+              title="When to use"
+              description={`
+              - Highlighting an item that floats above other content, such as temporary messages or cards
+              - IconButton and other elements, serving as an affordance for scrolling behaviors
+              `}
+            />
+            <MainSection.Card
+              cardSize="md"
+              type="don't"
+              title="When not to use"
+              description={`
+              - Content that doesn't need elevation or have its own setup (e.g. Pins, boards)
+              `}
+            />
+          </MainSection.Subsection>
+        </MainSection.Subsection>
+        <MainSection.Subsection
+          title="Raised"
+          description={`
+          Presents a drop shadow on the edge of a top or bottom component, allowing surfaces to move behind when scrolled.
+
+          ⚠️ Note: Raised shadow options will be available automatically in Box soon.
```

Comment:
Given this note, should the Floating option be called out that it is available on Box? Given the presentation of the token and values, it makes it seem that the user has to implement the shadow themselves using dangerouslySetInlineStyle